source/onboard/sso-saml.rst

```diff
@@ -80,16 +80,6 @@ Existing members that are identified by this attribute will be promoted from mem
 .. note:: 
     If the ``admin`` attribute is set to ``false`` the member's role as System Admin is retained. However if the attribute is removed/changed, System Admins that were promoted via the attribute will be demoted to members and will not retain access to the System Console. When this attribute is not in use, System Admins can be manually promoted/demoted in **System Console > User Management**.
 
-Roadmap
--------
-
-From Mattermost v5.14, you can optionally configure Mattermost to sign the SAML request using a private key to meet InfoSec requirements at your organization.
```

Comment:
Is there value in keeping the pieces that describe current functionality and remove only the line about future roadmap?

Reply:
Possibly, i think it's covered in the other docs. It might make more sense to move them to a Q/A then.

Something like:

```
> Can I provision and deprovision users who log in via SAML?

Yes, but this relies on AD/LDAP to do so. Currently, we do not support SCIM.
```

I swear we covered this somewhere else too, but i'm drawing a blank

Reply:
I'll dig in further across the docs set to see if we have duplicate content we can eliminate.

Reply:
I think it can probably live here - https://docs.mattermost.com/onboard/ad-ldap.html#how-do-i-deactivate-users and have a FAQ on on the SAML page if anyone asks where we can call our SCIM being unsupported.

Just means updating that ldap faq to also briefly talk about SAML and LDAP users getting deactivated

Reply:
This is great, @coltoneshaw! Can I leave this update with you?